Library/Homebrew/test/rubocops/caveats_cop_spec.rb

```diff
@@ -1,41 +1,20 @@
-require "rubocop"
-require "rubocop/rspec/support"
-require_relative "../../extend/string"
 require_relative "../../rubocops/caveats_cop"
 
 describe RuboCop::Cop::FormulaAudit::Caveats do
   subject(:cop) { described_class.new }
 
   context "When auditing caveats" do
     it "When there is setuid mentioned in caveats" do
-      source = <<~EOS
+      expect_offense(<<~EOS)
```

Comment:
why is this one EOS not RUBY?

Reply:
Must have gotten tired of copy-pasting. 😄